groclient/cli.py

```diff
@@ -1,13 +1,51 @@
+from random import random
 import argparse
 import getpass
 import os
 import sys
+import unicodecsv
 
 import groclient.lib
 import groclient.cfg
 from groclient import GroClient
 
-OUTPUT_FILENAME = "gro_client_output.csv"
+
+def pick_random_data_series(client):  # pragma: no cover
+    """Given a selection of entities, pick a random available data series the given selection
```

Comment:
We should update the description as we don't pass any selection

Reply:
Updated